client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -16,25 +17,41 @@ import PacmanBoard from './PacmanBoard';
 class PacmanGame extends Component {
   state = {
     pacman: {
-      x: 0,
+      x: 1,
       y: 5,
+      direction: 'RIGHT',
     },
-    direction: 'RIGHT',
+    ghosts: [
```

Comment:
These ghosts will be randomly generated. am i right?

Reply:
Yeah. but for now, they have a static starting point. Kaushik is working on giving them a random start position.